hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
This decorator is not required and in fact may be harmful by tricking that the linear binning works lazily (it does not).

I suggest removing it and creating a new issue regarding the lazy implementation